monitoring-service/src/main/java/org/terracotta/management/service/monitoring/MonitoringServiceProvider.java

```diff
@@ -42,7 +46,8 @@ public boolean initialize(ServiceProviderConfiguration configuration) {
     MonitoringServiceConfiguration config = configuration instanceof MonitoringServiceConfiguration ?
         (MonitoringServiceConfiguration) configuration :
         new MonitoringServiceConfiguration();
-    this.monitoringService = new MonitoringService(config);
+    SequenceGenerator generator = new BoundaryFlakeSequenceGenerator(TimeSource.BEST, NodeIdSource.BEST);
```

Comment:
I don't know... Here we could have an implementation that looks into the `config`. The config comes from the `tc-config.xml` file so I am wondering if we shouldn't add instead some XML attributes to customize which implementation to use ? I.e. BEST (default), MAC_PID, or a custom class set in tc-config.

what do you think ? we could have a special impl here that uses the content of the config file ?